src/java/org/apache/cassandra/index/sai/disk/v2/sortedterms/SortedTermsReader.java

```diff
@@ -105,19 +105,34 @@ public SortedTermsReader(@Nonnull FileHandle termsData,
 
     /**
      * Returns the point id (ordinal) of the target term or the next greater if no exact match found.
+     * @param term
+     * @return
+     */
+    public long getNextPointId(@Nonnull ByteComparable term)
```

Comment:
i don't think "next" is the right word for describing "at least" behavior, i.e. "next" implies that it performs > instead of >=

Reply:
How about `getCeilingForPointId`?

Reply:
I was thinking (with gpt4's help) of it along the lines of `Math.ceil`, which `Returns the smallest (closest to negative infinity) double value that is greater than or equal to the argument and is equal to a mathematical integer.`. I think ceiling technically captures what we're looking for, but it's not a typical usage, so I am fine choosing something else.

We also already have `getLastPointId` on the class, so we could use `getFirstPointId`, though those don't feel right either since they are ambiguous in terms of boundary inclusivity.

I am good with `getExactOrNextPointId` and `getExactOrPreviousPointId`. I'll update https://github.com/datastax/cassandra/pull/787

Reply:
you're right, ceiling and floor are correct.  wfm.